apple/internal/codesigning_support.bzl

```diff
@@ -239,6 +239,12 @@ def _should_sign_simulator_bundles(ctx):
       True/False for if the bundle should be signed.
 
     """
+    if "apple.codesign_simulator_bundles" in ctx.var:
+        print("warning: --define apple.codesign_simulator_bundles is deprecated, please switch to --features apple.skip_codesign_simulator_bundles")
+
+    if "apple.skip_codesign_simulator_bundles" in ctx.features:
```

Comment:
we want to use proper feature configuration, so not_features is also taken into account

Reply:
ha yep nice call, fixed I believe